Formula/sickbeard.rb

```diff
@@ -38,6 +38,10 @@ def install
     (bin+"sickbeard").write(startup_script)
```

Comment:
Not a big thing, but if you see these `(bin+"sickbeard").write(startup_script)` sorts of things with the `+` instead of `/` feel free to change it to a `/` where appropriate.